src/ralph/scan/plugins/idrac.py

```diff
@@ -335,6 +353,11 @@ def idrac_device_info(idrac_manager):
     fibrechannel_cards = _get_fibrechannel_cards(idrac_manager)
     if fibrechannel_cards:
         device_info['fibrechannel_cards'] = fibrechannel_cards
+    slotnumber = _get_enclosures(idrac_manager)
+    if slotnumber:
+        if (slotnumber[0] == '10'
+                and device_info['model_name'] == 'Dell PowerEdge R630'):
+            device_info.update({'model_name': 'Dell r630 - 10 disks slot'})
```

Comment:
we cannot accept it in current form - this is very custom case, probably used internally. Besides that you shouldn't rely on device model and use asset model instead.

Why do you want to extract this case as separated model?